git/provider_gitlab.go

```diff
@@ -191,46 +164,35 @@ func (p *GitLabProvider) getProjects(ctx context.Context, gl *gitlab.Client, r *
 		err      error
 	)
 	if !p.IsPersonal {
```

Comment:
Can you please create two function `getPersonalProjects` and `getGroupProjects`

Reply:
I split them into `getPersonalProject` and `getGroupProject`, as the return of the list when we are looking for a single item made little sense to me.